bundle/manifests/rhods-operator.clusterserviceversion.yaml

```diff
@@ -98,16 +99,9 @@ metadata:
     capabilities: Full Lifecycle
     categories: AI/Machine Learning, Big Data
     certified: "False"
-    containerImage: quay.io/opendatahub/opendatahub-operator:v2.0.0
-    createdAt: "2024-10-15T10:03:01Z"
-    features.operators.openshift.io/disconnected: "true"
-    features.operators.openshift.io/fips-compliant: "false"
-    features.operators.openshift.io/proxy-aware: "false"
-    features.operators.openshift.io/tls-profiles: "false"
-    features.operators.openshift.io/token-auth-aws: "false"
-    features.operators.openshift.io/token-auth-azure: "false"
-    features.operators.openshift.io/token-auth-gcp: "false"
-    olm.skipRange: '>=1.0.0 <2.0.0'
+    containerImage: REPLACE_IMAGE:latest
```

Comment:
i do not know who exactly devops replace scripts are doing for these now.
if they look for key name  or keyword
need double check with them

Reply:
> need double check with them

Yeah. I just get from 2.16.

Reply:
Keep it as is. The automation will take care of substituing REPLACE_IMAGE:latest with the right image reference